### ISSUE TITLE
Shenmue 2: Assertion failed!

### ISSUE THREAD

**Issue by satoshi213** [xemu-project/xemu#423]
### Title

https://xemu.app/titles/4d530032/#Shenmue-II

### Bug Description

Steps to reproduce:

    Boot game
    Start single player campaign (load save)

make the riddle of the keys in the statue
pass the animation
fight against the henchmen
pass the animation
leave the room walk to the stairs when going up
the error happens. 

Message:
microsoft visual c++ runtime library
assertion failed
program
file ../hw/xbox/nv2a/pgraph.c
expression (index_count*vertex_size)
pg-inline_array_length*4
for information on how your program can cause an 
assertion failure, see the visual c++ documentation on asserts

### Expected Behavior

go up the stairs and continue the game. 

### xemu Version

Version: 0.5.4-39-gedf3735718
Branch: master
Commit: edf3735
Date:   Tue Jul 20 17:39:54 UTC 2021

### System Information

OS: Windows 10.0.19043
CPU: Ryzen 3100
GPU: Nvidia Geforce GTX 950
GPU Driver: 471.41

**Comment by Triticum0** [comment]
@satoshi213 Could you be more specific I can't find where you're talking about a can't find it.

**Comment by Triticum0** [comment]
Could you post a save Please

**Comment by abaire** [comment]
@Triticum0 the assertion message got eaten in the original post, but I strongly suspect it was the same as #424 and PR #985 fixed this (so any release later than [v0.7.23](https://github.com/mborgerson/xemu/releases/tag/v0.7.23) should have the fix).

**Comment by Triticum0** [comment]
Should be fixed. someone should close

**Comment by abaire** [comment]
@Triticum0 can you confirm that this is fixed? Looks like you thought it was back in 2022, so it'd be good to close it out if it's no longer relevant.

**Comment by Triticum0** [comment]
Yeah I going close this issue If I understand correctly about 8 hours in this 12 hour game unless a save provied I leave this closed. for reference the crash would happen about here https://youtu.be/7V5L2u2P9RA?t=7352. Note: there are no save online. @satoshi213

**Comment by blakeplusplus** [comment]
@Triticum0 should be closed as invalid

**Comment by abaire** [comment]
I think this one isn't invalid, it was an actual issue that was (likely) fixed by #985 and ideally would still be verified as actually fixed with a recent xemu release build.
